.github/actions/javascript/getPreviousVersion/getPreviousVersion.ts

```diff
@@ -1,12 +1,17 @@
 import * as core from '@actions/core';
 import {readFileSync} from 'fs';
+import type {PackageJson} from 'type-fest';
 import * as versionUpdater from '@github/libs/versionUpdater';
 
 const semverLevel = core.getInput('SEMVER_LEVEL', {required: true});
 if (!semverLevel || !Object.values<string>(versionUpdater.SEMANTIC_VERSION_LEVELS).includes(semverLevel)) {
     core.setFailed(`'Error: Invalid input for 'SEMVER_LEVEL': ${semverLevel}`);
 }
 
-const {version: currentVersion} = JSON.parse(readFileSync('./package.json', 'utf8'));
-const previousVersion = versionUpdater.getPreviousVersion(currentVersion, semverLevel);
+const {version: currentVersion}: PackageJson = JSON.parse(readFileSync('./package.json', 'utf8'));
+if (!currentVersion) {
+    core.setFailed('Error: Could not read package.json');
+}
+
+const previousVersion = versionUpdater.getPreviousVersion(currentVersion ?? '', semverLevel);
```

Comment:
Is default value needed here? I thought the if above should be enough
```suggestion
const previousVersion = versionUpdater.getPreviousVersion(currentVersion, semverLevel);
```

Reply:
unfortunately not :/ ts is still complaining about the `undefined`

Reply:
I think we should put `return;` after `core.setFailed`, that makes sense and should resolve the problem, right?

Reply:
unfortunately not, again :(
```A return statement can only be used within a function body.```